mgmtsystem_audit/__manifest__.py

```diff
@@ -22,5 +22,5 @@
         "wizard/copy_verification_lines.xml",
     ],
     "demo": ["demo/demo_audit.xml"],
-    "development_status": "Production/Stable",
```

Comment:
Why did you remove this?

Reply:
When I remove this, the message become : 
"mgmtsystem_audit (Production/Stable) depends on mgmtsystem_nonconformity (Beta)"

That's why I had removed it.